src/br/analysis/run_features.py

```diff
@@ -0,0 +1,169 @@
+# Free up cache
+import argparse
+import os
+import sys
+
+import pandas as pd
+
+from br.analysis.analysis_utils import (
+    _get_feature_params,
+    _setup_evaluation_params,
+    _setup_gpu,
+    str2bool,
+)
+from br.features.plot import collect_outputs, plot
+from br.models.compute_features import compute_features
+from br.models.load_models import get_data_and_models
+from br.models.save_embeddings import save_emissions
+
+
+def main(args):
+    # Setup GPUs and set the device
+    _setup_gpu()
+    device = "cuda:0"
```

Comment:
This seems non-portable? Doesn't have to be fixed in this PR since we are already tracking this issue #9

Reply:
"cuda:0" as device should work if any gpu is visible. it will not work without gpu in which case we need to set device = 'cpu'